source/sayAllHandler.py

```diff
@@ -169,8 +169,7 @@ def _onLineReached(obj=self.reader.obj, state=state):
 			reason=controlTypes.REASON_SAYALL,
 			useCache=state
 		)
-		speechGen = speech.GeneratorWithReturn(speechGen)
-		seq = speech._flattenNestedSequences(speechGen)
+		seq = list(_flattenNestedSequences(speechGen))

```

Comment:
`_flattenNestedSequences` now returns a generator, so it can preserve generators.